lib/apiRouter.ts

```diff
@@ -13,7 +13,7 @@ export async function Router<T, D>(req: NextApiRequest, controller: controller):
                 await controller.Post(req.body as T)
                 break;
             case 'GET':
-                const result = await controller.Get(req.body as T);
+                const result = await controller.Get(req.query.id as string);
```

Comment:
Changed the parameter to be the id attribute in query params, as next.js doesn't accept body with GET requests. So, I am using the query id to get the job.

Reply:
GET, HEAD
both doesn't accept body
its not related to nextjs, its http standers

Reply:
you should take care, this will impact the Opportunity controller